{"uid":"fba5f71314d8e405","name":"test_chainId[Test for Tinkernet]","fullName":"tests.test_network_parameters.TestChainId#test_chainId","historyId":"e1a950524d3505597fedb3fb9c0502e6","time":{"start":1669436298037,"stop":1669436298762,"duration":725},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Tinkernet","statusTrace":"self = <tests.test_network_parameters.TestChainId object at 0x7f0acb01a100>\nchain = <tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>\n\n    def test_chainId(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Tinkernet\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Tinkernet","statusTrace":"self = <tests.test_network_parameters.TestChainId object at 0x7f0acb01a100>\nchain = <tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>\n\n    def test_chainId(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Tinkernet\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"5b95388c9e8e7114","name":"stdout","source":"5b95388c9e8e7114.txt","type":"text/plain","size":54}],"parameters":[],"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestChainId"},{"name":"host","value":"fv-az445-218"},{"name":"thread","value":"1655-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"fba5f71314d8e405.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f0acb0af7c0>"]}